.gitlab/e2e.yml

```diff
@@ -80,6 +80,9 @@ k8s-e2e-otlp-main:
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/test-infra-definitions/runner$TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX:$TEST_INFRA_DEFINITIONS_BUILDIMAGES
   tags: ["arch:amd64"]
   before_script:
+    # Setup DD API Key
+    - set +x
+    - export DD_API_KEY=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.datadog_api_key_org2 --with-decryption --query "Parameter.Value" --out text)
```

Comment:
I think we should use a secret param for it.

Reply:
Done, it will use the secret parameter by default. The key can be overriden with env variables for local tests for example